### DISCUSSION TITLE
ARIA

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#114] [State of HTML Features]
Attributes to make markup more accessible (`role`, `aria-label`, `aria-haspopup` etc).

```html
<div id="tab1" aria-controls="panel1" role="tab">Prices</div>
<div id="panel1" class="panel" aria-labelledby="tab1" role="tabpanel">       
	<!-- tab content -->
</div> 
```

### MDN URL:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

### Specification URL:

https://www.w3.org/TR/wai-aria-1.2/

### Any other links?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes

### How stable is this feature?

Stable (Widely supported for several years)

### Type of feature

HTML attribute

### What focus area does this feature fall into?

Accessibility

### Details

ARIA as a whole, including `role` and all the `aria-*` attributes.
Do we want to break this down? If so, how? See also, [this discussion](https://github.com/Devographics/surveys/discussions/115).

**Comment by hidde** [comment]
A few thoughts: 

- I think 'stable' / ‘widely supported’ is a bit tricky; there are parts of ARIA with very wide browser/AT support and parts with buggy, little, or no support. This is not necessarily widely known among developers, which has all sorts of effects on end users. [ARIA-AT](https://aria-at.w3.org/) aims to improve interop and recently resulted in adding test results to Authoring Practices. 
- you probably are aware, but ARIA is language independent, so also independent from HTML
- in a lot of cases, ARIA roles, states and properties also exist in HTML… I'm not sure what the best way would be to deal with this overlap in the survey?
